packages/core/__tests__/transform/rewrite.test.ts

```diff
@@ -467,6 +467,29 @@ describe('Transform: rewriteModule', () => {
   });
 
   describe('ember-template-imports', () => {
+    test('in class extends', () => {
+      let customEnv = GlintEnvironment.load(['ember-loose', 'ember-template-imports']);
+      let script = {
+        filename: 'test.gts',
+        contents: stripIndent`
+          import Component, { hbs } from 'special/component';
+          export default class MyComponent extends Component(<template></template>) {
```

Comment:
this is a goofy test, but the real-world use case is ember-route-template, yea? 

https://github.com/discourse/ember-route-template

Reply:
and https://github.com/patricklx/ember-routable-component?tab=readme-ov-file#usage